packages/telemetry/src/index.ts

```diff
@@ -1,21 +1,29 @@
+import openTelemetry from '@opentelemetry/api'
+import { detectResourcesSync, Resource } from '@opentelemetry/resources'
+import { SemanticResourceAttributes } from '@opentelemetry/semantic-conventions'
+import { OnStartSpanProcessor } from './processors/on-start-span-processor'
+import { ConsoleTraceLinkExporter } from './span-exporters/console-trace-link-exporter'
+
 import type { Span, SpanOptions, Tracer, Context, Attributes } from '@opentelemetry/api'
 import type { BasicTracerProvider, SimpleSpanProcessor, BatchSpanProcessor, SpanExporter } from '@opentelemetry/sdk-trace-base'
 import type { DetectorSync } from '@opentelemetry/resources'
+const types = ['child', 'root'] as const
 
-import openTelemetry/*, { diag, DiagConsoleLogger, DiagLogLevel }*/ from '@opentelemetry/api'
-import { SemanticResourceAttributes } from '@opentelemetry/semantic-conventions'
-import { Resource, detectResourcesSync } from '@opentelemetry/resources'
+export const enabledValues = ['true', '1']
 
-const types = ['child', 'root'] as const
+const SERVICE_NAME = 'cypress-app'
```

Comment:
```suggestion
const STAGING_ENV = 'cypress-app-staging'
const PRODUCTION_ENG = 'cypress-app`
```

Reply:
The service name remains the same regardless of staging or prod, we flex cloud endpoint we send data too to separate staging and production.

Reply:
oops yeah, this wasn't meant to replace the Service_name, more like should we pull this up so we can see face-up the names/envs we push data to.